Detectors/ITSMFT/MFT/macros/SetIncludePath.C

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Can you please avoid an anonymous macro here and make a function `void SetIncludePath()`. Unfortunately, on MacOs, interpreting this macro returns a nonzero exit code which our unit test considers a failure. We had fixed this in the original macro directory.